modules/core/src/main/scala/org/tessellation/http/routes/DagRoutes.scala

```diff
@@ -7,8 +7,9 @@ import cats.syntax.functor._
 import org.tessellation.dag.domain.block.DAGBlock
 import org.tessellation.domain.cell.{L0Cell, L0CellInput}
 import org.tessellation.domain.dag.DAGService
-import org.tessellation.ext.http4s.{AddressVar, SnapshotOrdinalVar}
-import org.tessellation.security.signature.Signed
+import org.tessellation.ext.http4s.SnapshotOrdinalVar
```

Comment:
Shouldn't `SnapshotOrdinalVar` also go to `schema` module?

Reply:
probably yes, I'll check

Reply:
done :+1: